introduction_to_amazon_algorithms/jumpstart-foundation-models/text2text-fine-tuning-flan-t5.ipynb

```diff
@@ -0,0 +1,736 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    model_list = [m for m in list_jumpstart_models(filter=filter_value) if "flan-t5" in m]
</pre><p>it will fail for XXL.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3916/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
there's a filter for "training_supported == true", currently xxl will not appear in the list